gitlab-pipeline/stage/release.yml

```diff
@@ -39,6 +39,7 @@ release:testing-boards:beagleboneblack:manual:
   dependencies:
     - init:workspace
     - build:acceptance:beagleboneblack
+    - trigger:integration-tests
```

Comment:
I understand your intentions but to force green integration tests to then release binaries stuff is tricky. When we talked about it I was referring only to the container images, not the other artifacts like here beagleboneblack, mender-cli, gateway, etc. There are many occasions that we want to publish these despite failing tests - for example release candidates.

But because I see the value in having "all green pipeline" as you kind of are expecting in this PR, I suggest removing the requirement for `manual` versions of the release jobs (so that a human can trigger things on his/her risk) but keep it in the automatic jobs as a requirement.